manifests/interface.pp

```diff
@@ -103,7 +107,7 @@
   $gateway         = undef,
   $hwaddr          = undef,
   $mtu             = undef,
-
+  $arp             = undef,
```

Comment:
Would you please move this param after the redhat specific ones (from line 181 https://github.com/Infraded/puppet-network/blob/ARP-patch/manifests/interface.pp#L181)?